doc/user/cluster_tls.md

```diff
@@ -27,6 +27,8 @@ spec:
       operatorSecret: operator-etcd-client-tls
 ```
 
+Note: This example cluster and example certs are provided in [example/tls](../../example/tls/).
```

Comment:
remove Note.

The example key/certs can be found in the [example/tls directory][aaa]